includes/features/SU-SWS/stanford_jumpstart_home/homepage-mayfield-dynamic.feature

```diff
@@ -3,13 +3,13 @@ Feature: Mayfield Homepage Layout
   As an end user
   I want to check for the existence of content that should appear
 
-  @api @dev @destructive @jsa
+  @api @dev @destructive
   Scenario: Set the homepage to Mayfield news and events
     Given I am logged in as a user with the "administrator" role
     And I am on "admin/stanford-jumpstart/customize-design"
     Then I press the "edit-layouts-stanford-jumpstart-home-mayfield-news-events-selector" button
 
-  @deploy @jsa @live @safe
+  @safe
   Scenario Outline: Header content for mayfield with news and events.
```

Comment:
I agree. These homepage tests were pretty buggy when I tried them out but it could be because they were @live instead of @deploy.  I think we can just write a general layout test that works across all sites when the site is @live.